website/source/docs/providers/vsphere/r/virtual_machine.html.markdown

```diff
@@ -68,6 +68,7 @@ The following arguments are supported:
 * `datacenter` - (Optional) The name of a Datacenter in which to launch the virtual machine
 * `cluster` - (Optional) Name of a Cluster in which to launch the virtual machine
 * `resource_pool` (Optional) The name of a Resource Pool in which to launch the virtual machine. Requires full path (see cluster example).
+* `host_system` (Optional) The name of a ESXi host in which to launch the virutal machine. Requires FQDN as presented in vCenter.
```

Comment:
in govmomi's object, it is being referred to as HostSystem, hence i decided to follow suite.